src/SMTLIB/Backends.hs

```diff
@@ -35,14 +46,13 @@ flushQueue q = atomicModifyIORef q $ \cmds ->
 -- a function for logging the solver's activity, and an optional queue of commands
 -- to send to the backend.
 --
--- A solver can either be in eager mode or lazy mode. In eager mode, the queue of
--- commands isn't used and the commands are sent to the backend immediately. In
--- lazy mode, commands whose output are not strictly necessary for the rest of the
--- computation (typically the ones whose output should just be "success") and that
--- are sent through 'command_' are not sent to the backend immediately, but
--- rather written on the solver's queue. When a command whose output is actually
--- necessary needs to be sent, the queue is flushed and sent as a batch to the
--- backend.
+-- A solver can either be in eager (non-queuing) mode or lazy (queuing) mode. In
+-- eager mode, the queue of commands isn't used and the commands are sent to the
+-- backend immediately. In lazy mode, commands whose output are not strictly
+-- necessary for the rest of the computation (typically the ones whose output should just be "success") and that are sent through 'command_' are not sent to the backend
```

Comment:
Done. By the way, is there a tool for aligning these comments automatically (`ormolu` doesn't seem to be doing that) ? It's tedious to realign everything when adding even a single word, and it would also enforce a fixed maximum line length which could otherwise depend on the width of the editor of the person that writes the comment.

Reply:
Nice !